docs/usage/release-notes-for-major-versions.md

```diff
@@ -7,6 +7,56 @@ The most recent versions are always at the top of the page.
+- **datasource/docker:** Docker Hub lookups prefers `hub.docker.com` over `index.docker.io`. To revert to the old behavior: set `RENOVATE_X_DOCKER_HUB_TAGS_DISABLE=true` in your env
+- **git:** check _all_ commits on the branch to decide if the branch was modified ([#28225](https://github.com/renovatebot/renovate/pull/28225))
+- **gitea:** use "bearer auth" instead of "token auth" to authenticate to the Gitea platform
+- **github:** if you run Renovate as a GitHub app then `platformCommit` is automatically enabled
+- **http:** remove `dnsCache`
+- **logging:** you must set file logging via env, not in `config.js`
+- **manager/pep621:** change `depName` for `pep621` dependencies. This also changes the branch name for `pep621` updates, which may cause Renovate to autoclose and re-open some `pep621` PRs. Further this roll changes into a single PR which have not been grouped before. 
+- **npm:** for npm versions lower than 7, drop support for remediating vulnerabilities in _transitive_ dependencies
+- **npm:** remove `RENOVATE_CACHE_NPM_MINUTES` ([#28715](https://github.com/renovatebot/renovate/pull/28715))
+- **packageRules:** `matchPackageNames` (and related functions) no longer fall back to checking `depName`
+- **packageRules:** `matchPackageNames` exact matches are now case-insensitive
+
+### Commentary for 38
+
+If you're self-hosting using Renovate's Docker `-slim` images, you need to now drop that suffix - Renovate's default tags such as `38.0.0` are "slim" by default.
+There's no change if you're using the `-full` images.
+
+If you're self-hosting without Renovate's Docker images (e.g. building your own image, or running from the `renovate` npm package), then note that Renovate now requires Node.js `^20.15.1` to run.
+We dropped Node.js 18 and do not yet support Node.js 22 as it's non-LTS and not recommended for production.
+We decided to require the current non-vulnerable Node.js release (`20.15.1`) or above although do not plan to bump this minimum Node.js v20 required version in any non-major releases.
+Please ensure you are running the a secure version of Node.js v20 in future as sometimes Node.js vulnerabilities can impact Renovate.
+
+We recommend all users running as a GitHub App utilize `platformCommit` so have refactored it to default to enabled whenever a GitHub App token is detected.
+For PATs, it's still recommend to use regular commits.
+
+File-based logging must be configured using environment variables (e.g. `LOG_FILE`) now and not in file or CLI (such as `logFile`).
+This to ensure that logging can begin at the start of Renovate execution and captures the parsing of configs too.
+
+Finally, we have consolidated may `matchPackage*` and `excludePackage*` options into `matchPackageNames` by enabling patterns.
+You can now do like `"matchPackageNames": "/^com.renovatebot/"` (regex) or `"matchPackageNames": "@renovate/*"` (glob) in addition to exact name matching.
```

Comment:
```suggestion
#### Changes to package matching

Finally, we merged the `matchPackage*` and `excludePackage*` options into `matchPackageNames`.
We also enabled patterns for the `matchPackageNames` config option.

This means you can now use regex or glob patterns:

- `"matchPackageNames": "/^com.renovatebot/"` (regex)
- `"matchPackageNames": "@renovate/*"` (glob)

And of course, you can still use exact name matching.
```

Changes:

- Add title
- Use list for regex and glob examples
- Use simple words
- Split long sentences